certification/internal/shell/less_than_max_layers_test.go

```diff
@@ -47,7 +47,7 @@ var _ = Describe("LessThanMaxLayers", func() {
 			BeforeEach(func() {
 				engine := fakeEngine.(FakePodmanEngine)
 				layers := make([]string, 50)
-				for i, _ := range layers {
+				for i := range layers {
```

Comment:
Below, purely a nit, more a learning opportunity for myself:

I read/heard/learned at some point (from a location that I can't remember now) that this kind of declaration makes it clear that the developers intentionally chose to ignore the value that would otherwise be returned by the right side of the equal sign. I agreed with the premise, though I see it as more valuable in functions that return multiple values where you could potentially not address the value you didn't need.

For this reason, I haven't really made an effort to change these linter complaints. What do you think?

Reply:
Considering this is in a test, I'm good with fixing this linter complaint. In other code, I might be more explicit.